src/library/Util.js

```diff
@@ -397,7 +396,11 @@ function objectArrayToArray(objArr) {
   const rows = [];
 
   // header row: just take the keys from the first row
-  const headers = keysFromObj(objArr[0], true);
+  // If the length of objArr is 0, indicate "No data"
+  let headers = ['No data'];
+  if (Array.isArray(objArr) && objArr.length > 0) {
+    headers = keysFromObj(objArr[0], true);
+  }
```

Comment:
Why do we want to return the string "No data" instead of just no data?

Reply:
Good question. It is because actually there is nothing in the cell. I expected a behavior of a table which has headers only when there is no data. `function objectArrayToArray(objArr)` generates columns based on API results so if there is no data it looks an error in the spreadsheet. I just felt to let user know what happened on no data.

- MBQUERY
- MBCUSTOMQUERY

_Data exist_
![image](https://user-images.githubusercontent.com/16159767/101585001-3bb5c780-3a22-11eb-97cf-cfc1998b05e7.png)

_No data (before)_
![image](https://user-images.githubusercontent.com/16159767/101584721-aadeec00-3a21-11eb-8cef-60905084e25b.png)

_No data (after)_
![image](https://user-images.githubusercontent.com/16159767/101585100-6e5fc000-3a22-11eb-9304-74a02c137f7e.png)

Reply:
Fair point. I think the “standard” for spreadsheets is either data, no data, or error. I think in this case let’s go with no data (empty cell) instead of the string “No data” or an error.